src/main/java/com/aws/greengrass/mqttclient/PublishRequest.java

```diff
@@ -13,7 +13,7 @@
 
 @Builder
 @Value
-public class PublishRequest {
+public class PublishRequest extends MqttRequest {
     @NonNull String topic;
```

Comment:
MqttRequest already has the field topic, so shouldn't need to declare it here again. Same for subscribe and unsubscribe